### DISCUSSION TITLE
Customize Go to Line and Find and Replace UI

### DISCUSSION THREAD

**Discussion by ManojkumarV** [microsoft/monaco-editor#3667] [Q&A]
Is it possible to customize Go to Line/ Find and Replace UI of monaco editor

**Comment by CareyJWilliams** [comment]
You can override some of their colours via a custom theme, e.g.: https://code.visualstudio.com/api/references/theme-color#editor-widget-colors

I don't believe there are any other "official" programmatic ways of customising these elements.
